# ---------------------------------------------------------
#           COLLISION AVOIDANCE CONFIGURATION FILE            
# ---------------------------------------------------------

#TOTAL number of robots
number_of_robots: 2

# NED origin definition
ned_origin_lat: 39.543330
ned_origin_lon: 2.377940

# Area of the square of exploration (m^2)
area_of_exploration: 1600

# Collision avoidance algorithm to use: {stop&wait, PF}
collision_algorithm: 'stop&wait'

# Separation between each horizontal path of the exploration
max_distance_between_paths: 5.0

# Stop and wait parameters

# Width of danger zone window = danger_zone_side/reduction_coefficient
reduction_coefficient: 3

# Potential Field parameters

# Critical distance
critical_distance: 7.0

# Proportional control value of yaw angular velocity
k_th: 0.5

# Goal weight (w1)
goal_weight: 1

# Obstacle weight (w2)
obstacle_weight: 2 